tests/test-timber.php

```diff
@@ -578,22 +539,6 @@ public function testPostsPerPageBig()
         $this->assertCount(15, $posts);
     }
 
-    /**
-     * @expectedDeprecated Timber\Timber::query_post()
-     */
-    public function testQueryPost()
```

Comment:
I guess this somehow tests whether `Timber::query_post()` would change the post global so that `get_the_ID()` returns the queried post. This logic is not supported anymore.